CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+-
+
+# v6.1.0
+
```

Comment:
Please make sure to mention that you upgraded Firestore. You don't need to mention the individual changes.

Reply:
done